templates/commands/goldentest/test_funcs.go

```diff
@@ -105,3 +113,71 @@ func parseTestConfig(path string) (*goldentest.Test, error) {
 	}
 	return test, nil
 }
+
+// getTempDir returns a temporary directory for golden tests usage.
+func getTempDir() (string, error) {
```

Comment:
Could we use os.MkdirTemp() for this? If not, could you add a comment explaining why?

Reply:
Oh nice suggestion! I simply don't know this func :)